eclipse-theia/theia
# repo: perform `yarn upgrade`

#### What it does



The pull-request performs a `yarn upgrade` of the repository in order to resolve known security vulnerabilities, and also reflect what downstream adopters will pull for fresh installs.

#### How to test



- confirm that CI passes (license:check action may take time so 3PPs are approved)
- confirm that `yarn audit` does not produce results

#### Follow-ups



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
